mainnet/erigon/erigon.toml

```diff
@@ -17,6 +17,7 @@
 "snapshots" = false
 "prune" = "htc"
 "private.api.addr" = "127.0.0.1:9098"
+"db.size.limit" = "8TB"
```

Comment:
@Wolmin Please add comment and let's document it @fhildeb in docs and `README.md` that validators need to be careful and common with this `Erigon` value. From `Erigon` we know that https://github.com/ledgerwatch/erigon/releases/tag/v2.55.0 (`8TB` is because of Ethereum Mainnet size probably, and not for archive node). We can keep it as it is.

Reply:
`Runtime limit of chaindata db size. You can change value of this flag at any time.` is description for `db.size.limit`Erigon flag. It should be 12TB by default in Erigon now: https://github.com/ledgerwatch/erigon/pull/8889/files

Reply:
> `Runtime limit of chaindata db size. You can change value of this flag at any time.` is description for `db.size.limit`Erigon flag. It should be 12TB by default in Erigon now: https://github.com/ledgerwatch/erigon/pull/8889/files

12TB breaks, I think its best if we stick with 8 - we can alter this if needed in the future, but I think 8TB is sufficient